src/cli.rs

```diff
@@ -37,6 +33,21 @@ pub struct PassedThroughArgs {
     pub args: Vec<OsString>,
 }
 
+// Generate passed thro args from `env::args_os()`. `PnpmCommand` can be formatted to string
+// to get the command name, so all we need is the remaining args. This is better than maintaining
+// a `PassedThroughArgs` struct for all variants of `PnpmCommand` and also wouldnt bloat the
+// size of the enum
+impl std::default::Default for PassedThroughArgs {
+    fn default() -> Self {
+        let mut os_args = std::env::args_os();
+        os_args.next(); // skip the bin name
+        os_args.next(); // skip the command name
+        Self {
+            args: os_args.collect(),
+        }
+    }
+}
+
```

Comment:
I am surprised `PassedThroughArgs` is still necessary. In any case, why `Default` and not `from_args(args: [OsString])`?

Reply:
passed through args is no longer needed. We can just discard it and pass os_args directly. as for why default, idk i forgot about from. if u say to discard passthroughargs, i can refactor the code to no longer use it. otherwise ill use `from::From` or maybe just write a method on it. your call.